index.d.ts

```diff
@@ -104,6 +104,25 @@ export type BandSchema = {
   maxValue?: number;
 };
 
+/**
+ * An object containing scales: A scale is defined by a label (usually a zoom level
+ * or scale denominator) and a scale denominator.
+ *
+ * Examples : {
+ *  0: 559082264.028717,
+ *  1: 279541132.014358
+ *  ...
+ * }
+ * or
+ * {
+ *    500: 500,
+ *    1000: 1000
+ *    ...
+ * }
+ *
+ */
+export type Scales = Record<number, number>;
```

Comment:
If the key of a scale is the label, it should probably be a string instead of a number, right?